client/scss/components/_button.scss

```diff
@@ -164,7 +164,9 @@
 
       .icon {
         @include svg-icon(1rem);
-        padding: 0.75em;
+        position: absolute;
```

Comment:
Nice work @Marmik2003! 

**Suggestion:** We could also get away without positioning absolute on this one.
The issue seems to only be happening in elements wrapped with `.icon-wrapper` above. If we add some flex-box centering to that instead it should also resolve the issue. 

e.g 
```
.icon-wrapper {
 display: flex;
 justify-content: center;
 align-items: center;
}
```